src/main/kotlin/org/ghrobotics/frc2019/Constants.kt

```diff
@@ -0,0 +1,18 @@
+package org.ghrobotics.frc2019
+
+import com.revrobotics.CANSparkMaxLowLevel
+import org.ghrobotics.lib.mathematics.units.inch
+import org.ghrobotics.lib.mathematics.units.nativeunit.NativeUnitLengthModel
+import org.ghrobotics.lib.mathematics.units.nativeunit.nativeUnits
+
+object Constants {

```

Comment:
Was there a reason for moving all of this to a separate constants file instead of keeping it at the top of Drivetrain.kt. Just interested in the reasoning -- perhaps this might make the code cleaner?

Reply:
yes it was to keep the code clean and also just to develop good habits because doing that is a good idea for a competition robot